src/Analyzers/Core/CodeFixes/CodeFixesResources.resx

```diff
@@ -168,4 +168,7 @@
   <data name="Convert_type_to_0" xml:space="preserve">
     <value>Convert type to '{0}'</value>
   </data>
+  <data name="Make_all_suitable_declarations_of_0_partial" xml:space="preserve">
+    <value>Make all suitable declarations of '{0}' partial</value>
```

Comment:
Nit: i would just simplify this to eitehr `Make type 'partial'` or `Make {0} 'partial'`.   "Suitable" and "Declaration" don't really add anything here.